### PR TITLE
Fix Compile warning on Windows for SET_STRIDE

### PR DESCRIPTION
**Pull request by AidanHa**
Cast 1 to uint64_t

Fixes GH-5161

Signed-off-by: Aidan Ha <qbha@edu.uwaterloo.ca>

### CODE REVIEW

File: runtime/vm/createramclass.cpp [hunk 1]
```diff
@@ -2857,7 +2857,7 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 						J9ARRAYCLASS_SET_STRIDE(ramClass, J9_VALUETYPE_FLATTENED_SIZE(elementClass));
 					}
 				} else {
-					J9ARRAYCLASS_SET_STRIDE(ramClass, (1 << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
+					J9ARRAYCLASS_SET_STRIDE(ramClass, ((U_64) 1 << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
```
Comment (DanHeidinga) [inline runtime/vm/createramclass.cpp]: `(U_64) 1` -> `((U_64) 1)`

The cast needs the extra brackets

Comment (AidanHa) [reply runtime/vm/createramclass.cpp]: Fixed

File: runtime/vm/createramclass.cpp [hunk 2]
```diff
@@ -2857,7 +2857,7 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 						J9ARRAYCLASS_SET_STRIDE(ramClass, J9_VALUETYPE_FLATTENED_SIZE(elementClass));
 					}
 				} else {
-					J9ARRAYCLASS_SET_STRIDE(ramClass, (1 << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
+					J9ARRAYCLASS_SET_STRIDE(ramClass, (((U_64) 1) << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
```
Comment (DanHeidinga) [inline runtime/vm/createramclass.cpp]:
```suggestion
					J9ARRAYCLASS_SET_STRIDE(ramClass, (((UDATA) 1) << (((J9ROMArrayClass*)romClass)->arrayShape & 0x0000FFFF)));
```

Comment (AidanHa) [reply runtime/vm/createramclass.cpp]: `SET_STRIDE` already casts to `UDATA`:

```#define J9ARRAYCLASS_SET_STRIDE(clazz, strideLength) ((clazz)->flattenedClassCache) = (J9FlattenedClassCache*)(UDATA)(strideLength)```

Comment (DanHeidinga) [reply runtime/vm/createramclass.cpp]: The cast needs to occur before the shift, the one in the macro occurs after

Comment (AidanHa) [reply runtime/vm/createramclass.cpp]: Ah, that makes sense. Will commit the change

**Comment by DanHeidinga** [comment]
I'm going to merge this without another build to get the builds unbroken.

**Pull request merged**